### ISSUE TITLE
Initial value of a subscription is always []

### ISSUE THREAD

**Issue by hvdklauw** [roman01la/react-horizon#1]
This is all nice when you want to work with lists, but you try to use ```find``` to get a single object getting an array initially throws a propType validation error.

Maybe another parameter for the connect function? so you can do the following:

```
export default connect(Component, {
    subscriptions: {
       item: (hz, props) => hz('table').find({ id: props.params.id }),
   },
   initial: {
       item: {},
   },
});
```

**Comment by roman01la** [comment]
What about default value as `undefined`? `defaultProps` can be used to set initial value.

**Comment by hvdklauw** [comment]
Yeah, that's even better

**Comment by hvdklauw** [comment]
Does that work with the isRequired check?

**Comment by roman01la** [comment]
@hvdklauw Yes, it is still props. But if you have `isRequired` `propType` without `defaultProps` then you'll get a warning in console which tells that required prop wasn't passed.

**Comment by roman01la** [comment]
Fixed in 78d7ba167160624001f4f8654d241de8c0f53d75
